static/js/components/AutoComplete.js

```diff
@@ -445,7 +454,10 @@ class AutoComplete extends Component {
       requestsList = requestsList.slice(0, maxSearchResults);
     }
 
-    this.requestsList = requestsList;
+    // 'renderItem' in Material-UI's Menu is only passed an option object when it is called downstream.
+    // This line prepares that 'renderItem' function ahead of time with some values that all MenuItems need and can't
+    // be determined by the option object alone.
+    let preparedMenuItemRenderer = _.partial(menuItemRenderer, this.getMenuItemSettings());
```

Comment:
It looks like `getMenuItemSettings` is only called here, why not just define the necessary value locally?